sdks/go/client/multiple_nodes_client.go

```diff
@@ -184,10 +193,14 @@ func FetchWithMajority[T any](ctx context.Context, nodes []*T, fetchFunc func(*T
 					}
 
 				}
+			} else {
+				fmt.Printf("Encountered an error while attempting to fetch with majority.\n Error: %v \n", res.err)
+				errs = append(errs, res.err)
```

Comment:
should we also add `fmt.Printf("error: failed to normalize json value: %v, error: %v", res.value, err)` to the errors list?